plat/mediatek/mt6795/bl31.ld.S

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
So far we put ARM-TF in internal SRAM, but some platform's SRAM space is not enough. We place xlat_table and coherent data in another memory space. In that case, we need to modify bl31 linker script to put data in another memory region.

If ARM can provide a way for platform linker script cusomization, that will be good.